addons/2d-color-picker/paint-editor.js

```diff
@@ -4,7 +4,7 @@
 import { normalizeHex } from "../../libraries/common/cs/normalize-color.js";
 import RateLimiter from "../../libraries/common/cs/rate-limiter.js";
 
-export default async ({ addon, console, msg }) => {
+export default /** @param {Addon.Userscript} */ async ({ addon, console, msg }) => {
```

Comment:
put it on the line above because

- it looks better
- prettier might get confused if it's there
- it still works above

Reply:
It's ugly to start the file with a comment IMO :P 
Not the case with the file you commented this on, but will happen in most other files